config.go

```diff
@@ -193,6 +200,14 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// Verify actions time limit
+	if cfg.ActionsTimeLimit.Seconds() <= 0 {
+		str := "%s ActionsTimeLimit cannot be <= 0"
```

Comment:
Need to add a colon here to match other logs:

```suggestion
		str := "%s: ActionsTimeLimit cannot be <= 0"
```